db/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM postgres:9.4
+FROM postgres:9.5
```

Comment:
Also this should use alpine image - `9.4-alpine` https://github.com/docker-library/postgres/blob/6420763a8acd9ba12d34b2bd76e6d6927d2c36a5/9.4/alpine/Dockerfile